prometheus_client/core.py

```diff
@@ -395,6 +395,13 @@ def close(self):
 def _MultiProcessValue(_pidFunc=os.getpid):
     files = {}
     values = []
+
+    # pid needs to be evaluated here in a master process
```

Comment:
That's usage dependant, not all relevant systems have masters. I think it'll be fine not to have this comment at all.